codegen/service/service.go

```diff
@@ -207,9 +207,10 @@ type Service interface {
 }
 
 {{- if .Schemes }}
-// auther defines the authorization functions used by the service.
-type auther interface {
+// Auther defines the authorization functions to be implemented by the service.
+type Auther interface {
 	{{- range .Schemes }}
+	{{ printf "%sAuth should implement the authorization logic for the %s security scheme." .Type .Type | comment }}
```

Comment:
nitpick: could the comment be `%sAuth implements the authorization logic for the %s security scheme."`

Reply:
I changed the comment to your suggestion.

By the way, is there a place I could discuss some improvements with you before doing pull requests ? You mentioned a Slack but I didn't find the URL.

Reply:
Great thank you! The details on how to join the slack channel are on https://goa.design (search for "slack".  There's also a badge in the README).